package.json

```diff
@@ -33,6 +33,7 @@
     "commander": "^2.11.0",
     "dockerode": "^2.5.0",
     "fs-extra": "^4.0.2",
+    "grpc": "^1.16.1",
```

Comment:
Platform-specific dependencies should be moved to the appropriate `npm run` script.